lib/ProductOpener/Products.pm

```diff
@@ -1001,6 +1001,25 @@ sub compute_sort_keys ($product_ref) {
 	return;
 }
 
+sub created_function($mongodb_to_mongodb, $product_ref, $products_collection, $data_root, $path) {
+
+	# make sure nutrient values are numbers
+	ProductOpener::Products::make_sure_numbers_are_stored_as_numbers($product_ref);
```

Comment:
as this code is now in Products.pm, you can remove the "ProductOpener::Products::" part.

Reply:
Okay thank you @stephanegigandet